README.md

```diff
@@ -65,25 +65,29 @@ sudo yum install gcc-c++ automake libtool pkgconfig boost-devel \
 To install on a FreeBSD system, use
 
 ```sh
-pkg install devel/git devel/autoconf devel/automake devel/gmake devel/libtool \
+pkg install devel/git devel/cmake
```

Comment:
Yes, only build and runtime dependencies should go here, not dev dependencies. That being said, we do need to document the development dependencies somewhere (#182)